### ISSUE TITLE
Set alert

### ISSUE THREAD

**Issue by KoIIIeY** [laravel-notification-channels/apn#121]
How can I set 'alert' value ?

**Comment by dwightwatson** [comment]
I’m not sure. Is it supported by the pushok library that we use under the hood? If so, I’d accept a PR that adds support for it.

**Comment by KoIIIeY** [comment]
As I see, Pushok don't have support for it.
https://pub.dev/packages/flutter_ios_voip_kit - flutter wants data like this:
`
{
    "aps": {
        "alert": {
          "uuid": <Version 4 UUID (e.g.: https://www.uuidgenerator.net/version4) >,
          "incoming_caller_id": <your service user id>,
          "incoming_caller_name": <your service user name>,
        }
    }
}
`

But I cant set custom alert field.

**Comment by dwightwatson** [comment]
I’m not familiar with that exact property but if it’s not available in pushok we won’t be able to support it.

**Comment by KoIIIeY** [comment]
In ApnAdapter.php every time creates new Alert, but Pushok allows use json string here. 
I see two ways to get right result:

1) Create custom alert string and set it to Pushok in adapt method.
2) Change Pushok code to allow merge custom values as arrays

**Comment by KoIIIeY** [comment]
@dwightwatson plz check this - https://github.com/laravel-notification-channels/apn/pull/122

**Comment by dwightwatson** [comment]
Available in v3.4.0.

**Comment by KoIIIeY** [comment]
Flutter example, working

```
public function toApnVoip($notifiable)
    {
        return ApnVoipMessage::create()
            ->setCustomAlert([
                'incoming_caller_id' => (string)$this->card->card_id,
                'incoming_caller_name' => $this->card->fullName,
                'uuid' => (string) \Str::uuid()
            ])
            ->badge(1);
    }
```
